src/mudlet.pro

```diff
@@ -662,6 +664,21 @@ linux|macx|win32 {
     }
 }
 
+linux {
+    HEADERS += ../3rdparty/discord/discord-rpc-linux/discord-rpc/linux-dynamic/include
```

Comment:
Are the other two paths correct - they look different to this one in that they are missing a `/discord` element straight after the `../3rdparty` directory:
`../3rdparty/discord/discord-rpc-linux/discord-rpc/linux-dynamic/include`
verses:
`../3rdparty/discord-rpc-osx/discord-rpc/osx-dynamic/include`
`../3rdparty/discord-rpc-win/discord-rpc/win32-dynamic/include`

Reply:
Thanks for the catch :)